ci/kind/test-e2e-kind.sh

```diff
@@ -55,6 +55,7 @@ THIS_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
 TESTBED_CMD="$THIS_DIR/kind-setup.sh"
 YML_CMD="$THIS_DIR/../../hack/generate-manifest.sh"
 FLOWAGGREGATOR_YML_CMD="$THIS_DIR/../../hack/generate-manifest-flow-aggregator.sh"
+SFTP_DEPLOYMENT_YML="$THIS_DIR/../../hack/externalnode/sftp-deployment.yml"
```

Comment:
maybe let's choose a better location for this file now that we are using it for an additional purpose
cc @luolanzone because it's related to a discussion we are having in #6234

Reply:
removed this part.